tasks.py

```diff
@@ -59,8 +59,9 @@ def process_send_notifications():
         if BCC_EMAIL_ADDRESSES:
             bcc_adresses = BCC_EMAIL_ADDRESSES.split(',')
 
-        if bericht.get('emailBehandelaar', {}).get('value'):
-            bcc_adresses.append(bericht.get('emailBehandelaar', {}).get('value'))
+        if bericht.get('typecommunicatie').get('value') != "Omzendbrief":
```

Comment:
can't we work with the URI's?

Reply:
Can't find `<https://kalliope.abb.vlaanderen.be/ld/algemeen/dossierType/omzendbrief>` in the db, we do store the typecommunicatie but not the dossierType afaik

Reply:
Only performing an api call to get these but in this service https://github.com/lblod/berichtencentrum-sync-with-kalliope-service/blob/master/kalliope_adapter.py#L102

Reply:
Ignore that. It is going to have too much impact to do it properly right now. Just make it case insensitive comparison